dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -85,7 +85,7 @@
 		adsp-clkctl-clk-ipll = <2>;
 		adsp-clkctl-freq-enc = <0xc 0x0 0x4>;
 		adsp-clkctl-freq-mask = <0x0 0x0 0x0>;
-		adsp-clkctl-freq-default = <2>;
+		adsp-clkctl-freq-default = <0>;
```

Comment:
This is interesting, not sure what should be the default policy. We used to run at the highest clock by default previously, why the change now?

Reply:
Good question. It turns out that in fact the boot process should take place at the highest clock and when everything is ready, we move to the lowest clock.